src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,28 @@
+package calculator;
+
+import calculator.Accumulator.AccumulatorController;
+
+public class Calculator {
+    
+    private final StringInput stringInput;
+    private int result;
+    
+    public Calculator(StringInput stringInput) {
+        this(stringInput, 0);
+    }
```

Comment:
```suggestion
    public Calculator(String text) {
        this(new StringInput(text), 0);
    }

    public Calculator(StringInput stringInput) {
        this(stringInput, 0);
    }
```
String을 받는 부 생성자가 있다면 Calculator 생성할 때 좀 더 편리하지 않을까?